tests/test_utils.py

```diff
@@ -1,130 +1,130 @@
-from __future__ import absolute_import, unicode_literals
```

Comment:
You might need to update your IDE. There is no real changes in this file:
<img width="2554" alt="image" src="https://user-images.githubusercontent.com/1629945/128811093-7574abec-baa6-44c0-887b-97bd094eab79.png">

Reply:
Yes, this is an IDE/OS issue. I've reverted the file.